src/styles/globals/_breakpoints.scss

```diff
@@ -6,5 +6,5 @@ $breakpoints: (
   // tablets
   'lg': 1280px,
   // desktops
-  'xl': 1920px,
+  'xl': 1921px,
```

Comment:
For some reason the paddings and margins were acting weird on my desktop, and this fixed it. But still if i zoom out the website, the bug appears again
![image](https://user-images.githubusercontent.com/43473505/128748919-89f895c9-f486-4116-84a9-dfce39a77412.png)
![image](https://user-images.githubusercontent.com/43473505/128748940-327def3a-b272-4229-8bec-a5f9d3ec4cdd.png)